controllers/index.js

```diff
@@ -23,6 +23,11 @@ exports.index = (req, res) => {
             const user = gitData[1];
             const prs = [];
 
+            if (user.data.type !== 'User') {
+                deferred.reject('not user');
```

Comment:
Not that it really matters, but can we use `notUser` just find whitespace can sometimes make things a bit confusing.

Reply:
Done.